Update procedure on HAOS installation on x86_64 (#21554)
home-assistant/home-assistant.github.io: source/_includes/installation/operating_system.md

<b>Summary</b>

You will need to write the HAOS (Home Assistant OS) disk image directly to your boot media, and configure your x86 to use UEFI boot mode when booting from this media.
</div>

> ### Configure the BIOS on your x86-64 hardware

Reviewer: ```suggestion
### Configure the BIOS on your x86-64 Hardware
```

> ### Configure the BIOS on your x86-64 Hardware

Reviewer 2: I know this was suggested as a correction above, but it is a normal word and isn't capitalized in this context.

```suggestion
### Configure the BIOS on your x86-64 hardware
```

To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Secure Boot disabled. The following screenshots are from a 7th generation Intel NUC system. The BIOS menu will likely look different on your systems. However, the options should still be present and named similarly.

1. To enter the BIOS, start up your x86-64 hardware and repeatedly press the `F2` key (on some systems this might be `Del`, `F1` or `F10`).
![Enter BIOS using F2, Del, F1 or F10 key](/images/installation/intel-nuc-enter-bios.jpg)

1. Make sure the UEFI Boot mode is enabled.
![Enable UEFI Boot mode](/images/installation/intel-nuc-uefi-boot.jpg)

1. Disable Secure Boot.
![Disable Secure Boot mode](/images/installation/intel-nuc-disable-secure-boot.jpg)

1. Save the changes and exit.

- The BIOS configuration is complete.

> As a next step, we need to write the HA Operating System image to the target boot medium. The HAOS has no integrated installer. This means, the Operating System is not copied automatically to the internal disk.

Reviewer 2: ```suggestion
As a next step, we need to write the Home Assistant Operating System image to the target boot medium. The HAOS has no integrated installer. This means the Operating System is not copied automatically to the internal disk.
```

> - The "boot medium" is the medium your x86-64 hardware will boot from when it is running Home Assistant.
> - Typically, an internal medium is used for the x86-64 hardware. Internal media are S-ATA hard disk, S-ATA SSD, a M.2 SSD or a non-removable eMMC.
> - Alternatively, an external medium can be used to boot HAOS such as an USB SDD (not recommended).

Reviewer 2: ```suggestion
- The "boot medium" is the medium your x86-64 hardware will boot from when it is running Home Assistant.
- Typically, an internal medium is used for the x86-64 hardware. Examples of internal media are S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC.
- Alternatively, an external medium can be used to boot HAOS such as a USB SDD (not recommended).
```

> To install the HAOS internally on your x86-64 hardware, there are 2 methods:
> 
> 1. Copying the HAOS disk image from your Desktop computer onto your boot medium (e.g. by using a USB to S-ATA adapter). This is not an option for a non-removable eMMC on your x86-64 hardware, of course.
> To use this method, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
> 2. Copying a live operating system (e.g. Ubuntu) onto a USB device. Then, insert this USB device to your x86-64 hardware and start the Ubuntu.

Reviewer 2: ```suggestion
To install the HAOS internally on your x86-64 hardware, there are 2 methods:

1. Copying the HAOS disk image from your Desktop computer onto your boot medium (e.g. by using a USB to S-ATA adapter). This is not an option for a non-removable eMMC on your x86-64 hardware, of course.
To use this method, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
2. Copying a live operating system (e.g. Ubuntu) onto a USB device. Then, insert this USB device into your x86-64 hardware and start the Ubuntu.
```

### Start up your {{site.installation.types[page.installation_type].board}}

{% if page.installation_type == 'generic-x86-64' %}

> 1. If you used your Desktop system to write the HAOS your boot media, install the boot media ({{site.installation.types[page.installation_type].installation_media}}) in the generic-x86-64 system.

Reviewer 2: ```suggestion
1. If you used your desktop system to write the HAOS your boot media, install the boot media ({{site.installation.types[page.installation_type].installation_media}}) in the generic-x86-64 system.
```

> - If you used a live operating system (e.g. Ubuntu), shutdown the live operating system and make sure to remove the USB flash drive you used for the live system.

Reviewer 2: ```suggestion
- If you used a live operating system (e.g. Ubuntu), shut down the live operating system and make sure to remove the USB flash drive you used for the live system.
```
